SQL/Archive/17.1/2017-08-24_dicom_archive_help_text.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO `help` (parentID, hash, topic, content, created, updated) VALUES ((SELECT helpID FROM (SELECT * FROM `help`) as temp WHERE hash=md5('dicom_archive')), md5('tarchive_metadata'), 'DICOM Archive: Tarchive Metadata', 'The “View Details” page of the DICOM Archive displays metadata for a given DICOM tarchive (compressed package of DICOM files).\r\nAll DICOM header fields and other metadata from the DICOM dataset are presented in the page, including information about the candidate, number of acquisitions, where the files were acquired, and all files contained within the given archive.\r\n\r\nClicking on the Acquisition ID number (in blue text) will bring users to the MRI Violated Scans module displaying any issues encountered for this scan regarding protocol checks or data integrity.\r\nUsers can easily view the history of protocol check violations or data loading concerns associated with this scan.','2017-08-24 00:00:00',NULL);

```

Comment:
why is this in the table and not the markdown help subdirectory since the dicom archive supports it (and in fact, uses it below..)

Reply:
@llevitis ping